packages/svelte/src/runtime/action/public.d.ts

```diff
@@ -74,3 +74,5 @@ export interface Action<
 // Implementation notes:
 // - undefined extends X instead of X extends undefined makes this work better with both strict and nonstrict mode
 // - [X] extends [never] is needed, X extends never would reduce the whole resulting type to never and not to one of the condition outcomes
+
+export * from './index.js';
```

Comment:
this looks like an empty file. maybe we should remove this line and the file?

https://github.com/sveltejs/svelte/blob/0a08f21aa89d5a4916f4debba13c8b32fbe87b75/packages/svelte/src/runtime/action/index.js

Reply:
yeah, done